R/positioning.functions.R

```diff
@@ -147,7 +147,7 @@ direct.label <- structure(function # Direct labels for color decoding
   print(direct.label(ratplot))
   ## To put them on the same side, just manually specify the
   ## Positioning Method.
-  print(direct.label(ratplot,"last.qp")) 
```

Comment:
Reverting this edit was un-savable in RStudio, but I just found the option in Tools > Project Options > Code Editing, unchecked the 'Strip trailing horizontal whitespace when saving' option - pushing the final commit in a minute